### DISCUSSION TITLE
compatibility with R7-7840HS

### DISCUSSION THREAD

**Discussion by Johnson847** [DavidS95/Smokeless_UMAF#38] [Q&A]
Hello,
I'm using a laptop with R7-7840HS and no independent graphics card on Win11. Is this program compatible with the latest Zen4 CPUs (or 780M to be more specific)?

**Comment by algo23-Shuairui** [comment]
yes, I am also using 7840hs on Win11, but vram was limited to 512mb by HP bios, need help.

**Reply by Johnson847** [reply]
机械革命，很绝望，Radeon Software装不上，只能用oem出厂自带的驱动，核显负载一上去就崩溃，AMD和机革两边都找过了，无解

**Reply by jpower123** [reply]
Same here, hoping to go to 4gb vram on a xiaomibook with r7 7840hs.

**Reply by Johnson847** [reply]
True, this thing is literally killing me here. Can't use it for a day without crashings.

**Reply by Imagium719** [reply]
我也机革无界14+，很绝望

**Comment by lenielsen** [comment]
This issue seems to affect a lot of HP devices. I tested the regular version and the Beta with a Probook (Ryzen 7 5825U), an Elitebook with R7 7840U. No settings, only wifi + network.

```
Welcome to SREP (Smokeless Runtime EFI Patcher) 0.1.4c
Opened SREP_Config

Config Size: 0x4E
Parsing Config

Stripping NewLine, Carriage and tab Return

Current Parsing Op LoadFromFS
OP Detected

Commnand LoadFromFS 
Current Parsing SuppressIFPatcher.efi
Found File SuppressIFPatcher.efi 
Current Parsing Op Exec
OP Detected

Commnand Exec 
Current Parsing Op LoadFromFS
OP Detected

Commnand LoadFromFS 
Current Parsing Loader.efi
Found File Loader.efi 
Current Parsing Op Exec
OP Detected

Commnand Exec 
Executing Load from FS

Loaded Image Success -> EAC0000
EXEC %r

Executing Load from FS

Loaded Image Success -> EABF000
EXEC %r

```

**Reply by Johnson847** [reply]
Umm, can you please ELI5? What issue are you referring to?

**Reply by lenielsen** [reply]
https://github.com/DavidS95/Smokeless_UMAF/discussions/36

**Comment by Neurrone** [comment]
@gLarsVegas I'm curious if its possible to change the amount of VRAM for your Elitebook G10 with this tool? I know HP has their bios locked down, so am wondering whether changes via Smokeless_UMAF would work.

**Reply by lenielsen** [reply]
No, Smokeless does not present any advanced options on this device. Same goes for the G9 Generation (I tested a probook with Ryzen 7 5825U)

**Reply by Neurrone** [reply]
@gLarsVegas thanks for letting us know. I guess that also applies to the beta version?

I have an Elitebook 845 G10 coming in a couple of weeks, so was really hoping to be able to get more out of the device.

**Comment by Maxim766** [comment]
Good day! Continuing the conversation about laptops with phoenix processors (including the 7840hs), thanks for smokeless UMAF software, but will it be possible to change the iGPU and FCLK frequencies in the future? Do you need any additional data? 

My laptop model is: HP Pavilion 14 plus (14-ey0031AU)

**Comment by joaquinvacas** [comment]
Is there any kind of news regarding this topic? Elitebook 845 G10 7840U here, it seems it's possible to set 4GB VRAM on the BIOS but I would love to restrict even 16GB :)

**Reply by trevor-harte-ase** [reply]
I have a HP zBook Firefly G10 with 7840HS and maxxed it out with 96gb of DDR5.
The HP/AMD software allows me to select 'gaming' optimisation that lifts the VRAM from 2G to 8G, but that is all I can get dedicated.
I also cannot get the AMD CBS menu in the Device Manager area of Smokeless_UMAF.
It's annoying as i'd love to bump it up higher so that I can get larger LLM to load. I can load a 7B LLM into VRAM, or a 14B LLM with some CPU/shared resource, but I am unable to load 32B models, etc.
I'm hoping the G11 with the 8840 (which is basically same as 7840+betterNPU) will have a bios option to allow more VRAM for processing LLM, or why do they even bother adding NPUs...

**Reply by joaquinvacas** [reply]
Wow, hos did you get 8GB?
 Mine maxes at 4GB VRAM (Ihave 32GB installed right now)

Smokeless UMAF is empty for me as well, cannot access CBS.

**Reply by trevor-harte-ase** [reply]
The zBook bios options are 512mb, Auto and Gaming. When on auto it allocated 2Gb dedicated to graphics. when set to gaming it has set 8Gb dedicated to graphics. It seems to be the way the bios was designed for the zBook to get more from the 780m?
Note that the Elitebook as 7840U while the zBook has 7840HS. Not sure what difference that makes to the embedded graphics.

The zBook ultra G1a has bios options to add much more VRAM, but thats the latest (expensive) model with the Ryzen AI Max processor. Although I wish HP could be convinced to backport that bios setting to the G10.

**Comment by joaquinvacas** [comment]
AFAIK 7840HS is just a higher TDP version of the 7840U

I have the same BIOS settings but gaming optimized sets to 4GB VRAM.

Maybe it's something related to the amount of installed RAM (?)
